docs/source/intro/installation.rst

```diff
@@ -1,16 +1,892 @@
+* `RabbitMQ`_ (A message broker necessary for AiiDA to communicate between processes)
+
+Depending on your set up, there are a few optional dependencies:
+
+* `git`_ (Version control system used for AiiDA development)
+* `graphviz`_ (For plotting AiiDA provenance graphs)
+
+.. _graphviz: https://www.graphviz.org/download
+.. _git: https://git-scm.com/downloads
+.. _python: https://www.python.org/downloads
+.. _python3-pip: https://packaging.python.org/installing/#requirements-for-installing-packages
+.. _virtualenv: https://packages.ubuntu.com/bionic/virtualenv
+.. _virtualenvwrapper: https://packages.ubuntu.com/bionic/virtualenvwrapper
+.. _postgresql: https://www.postgresql.org/downloads
+.. _RabbitMQ: https://www.rabbitmq.com/
+
+AiiDA has been tested on the following platforms:
+
+* Ubuntu 14.04, 16.04, 18.04
+* Mac OS X
+
+We expect AiiDA to also run on:
+
+* Older and newer Ubuntu versions
+* Other Linux distributions
+* Windows Subsystem for Linux (WSL)
+
+Below, we provide installation instructions for a number of operating systems.
+
+.. div:: dropdown-group
+
+   .. dropdown:: Ubuntu
+
+      .. code-block:: console
+
+         $ sudo apt-get install \
+            postgresql postgresql-server-dev-all postgresql-client rabbitmq-server \
+            git python3-dev python3-pip
+         $ pip install aiida-core
+
+      See :ref:`Ubuntu instructions<intro:install:prerequisites:ubuntu>` for details.
+
+   .. dropdown:: MacOS X (Homebrew)
+
+      .. code-block:: console
+
+         $ brew install postgresql rabbitmq git python
+         $ brew services start postgresql
+         $ brew services start rabbitmq
+         $ curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+         $ python get-pip.py
+         $ pip install aiida-core
+
+      See :ref:`MacOS X (Homebrew) instructions<intro:install:prerequisites:brew>` for details.
+
+   .. dropdown:: MacOS X (MacPorts)
+
+      .. code-block:: console
+
+         $ sudo port install postgresql postgresql-server rabbitmq-server
+         $ pg_ctl -D /usr/local/var/postgres start
+         $ curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+         $ python get-pip.py
+         $ pip install aiida-core
+
+      See :ref:`MacOS X (MacPorts) instructions<intro:install:prerequisites:macports>` for details.
+
+   .. dropdown:: Gentoo Linux
+
+      .. code-block:: console
+
+         $ emerge -av git postgresql rabbitmq-server
+         $ rc-update add rabbitmq
+         $ emerge --ask dev-lang/python:3.7
+         $ emerge --ask dev-python/pip
+         $ pip install aiida-core
+
+      See :ref:`Gentoo Linux instructions<intro:install:prerequisites:gentoo>` for details.
+
+   .. dropdown:: Windows Subsystem for Linux
+
+      .. code-block:: console
+
+         $ sudo apt-get install \
+            postgresql postgresql-server-dev-all postgresql-client \
+            git python3-dev python-pip
+         $ sudo service postgresql start
+         # install rabbitmq on windows
```

Comment:
This should probably be mentioned in the description rather than a comment here.

Reply:
Fixed in 0cf7a7d